app/controllers/brexit_checker_controller.rb

```diff
@@ -52,12 +62,47 @@ def confirm_email_signup
 
   def save_results; end
 
+  def saved_results
+    redirect_to transition_checker_new_session_path and return unless logged_in?
```

Comment:
Should visiting this page when not logged in take you through the login flow and then redirect back to the results page, rather than the questions page?

Reply:
If it did the account manager could just link to `www.gov.uk/transition-check/saved-results` and not need to worry about constructing a querystring.

Reply:
I've condensed this into the `/login` flow, so after logging in, the user will be taken to either the questions or results (depending on whether they have anything saved).  In the account manager's "service card", we can just link to `/login`.

Reply:
<img width="225" alt="Screenshot 2020-10-07 at 13 48 24" src="https://user-images.githubusercontent.com/75235/95332792-d0296f80-08a3-11eb-8456-592f03b7330e.png">

I think we do want to be able to link to both independently

Reply:
I wonder if we'll need to add another `redirect_uri` for the client in the account manager then?  Since we've only got the `/login/callback` route at the moment, and this doesn't have any knowledge of the task you're trying to complete.

Reply:
I think we can use the `state` parameter we pass as part of the OAuth call for this.  The server passes it back unchanged.  Currently we're just using it to store the nonce, but it could be something like `{nonce}:q` or `{nonce}:r`

Reply:
Thanks for the pointer @barrucadu.  I've used the `state` to determine where to send the user after the callback.  Also added a `/edit-saved-results` route which can be used on the "Update answers" link.